src/main/resources/export/cpp/RobotContainer.cpp

```diff
@@ -0,0 +1,80 @@
+#header()
+
+\#include "RobotContainer.h"
+\#include <frc2/command/button/JoystickButton.h>
+
+#@autogenerated_code("includes", "")
+#parse("${exporter_path}OI-includes.h")
+#end
+
+#set($cmd = $robot.getProperty("Autonomous Command").getValue())
+
+#foreach ($component in $components)
+#if ($helper.exportsTo("OI", $component)
+     && "#type($component)" != "" 
+     && "#type($component)" == "Joystick")
+    #set($joy = $component)
+
+#end
+#end
+
+RobotContainer::RobotContainer() : m_autonomousCommand(
+#@autogenerated_code("constructor", "    ")
+#foreach( $component in $components )
```

Comment:
needs to be in a file with parse

Reply:
The code should also be similar to what's in RobotContainer.h and only do a subsystem once